.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+  {
+    "version": "0.1.0",
+    "configurations": [
+        {
+            "name": "pinetime Debug",
+            "type":"cortex-debug",
+
+            "cortex-debug.armToolchainPath":"/opt/gcc-arm-none-eabi-9-2020-q2-update/bin",
+            "cwd": "${workspaceRoot}",
+            "executable": "${command:cmake.launchTargetPath}",
+            "request": "launch",
+            "servertype": "external",
+            // This may need to be arm-none-eabi-gdb depending on your system
+            "gdbPath" : "/opt/gcc-arm-none-eabi-9-2020-q2-update/bin/arm-none-eabi-gdb",
```

Comment:
I should consider using environment variables for these to make it more portable for those who do not want to use docker.